elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/MetaDataStoreTransaction.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.metadata;
+
+import com.yahoo.elide.core.DataStoreTransaction;
+import com.yahoo.elide.core.RequestScope;
+import com.yahoo.elide.core.datastore.inmemory.HashMapDataStore;
+import com.yahoo.elide.core.filter.expression.FilterExpression;
+
+import com.yahoo.elide.request.EntityProjection;
+import com.yahoo.elide.request.Relationship;
+import com.yahoo.elide.request.Sorting;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * MetaDataStore transaction handler.
+ */
+public class MetaDataStoreTransaction implements DataStoreTransaction {
+    private final Map<String, HashMapDataStore> hashMapDataStores;
+
+    public MetaDataStoreTransaction(Map<String, HashMapDataStore> hashMapDataStores) {
+        this.hashMapDataStores = hashMapDataStores;
+    }
+
+    @Override
+    public void flush(RequestScope requestScope) {
+        // Do nothing
+    }
+
+    @Override
+    public void save(Object object, RequestScope requestScope) {
+        // Do nothing
```

Comment:
Maybe throw a 403 error (HttpStatusException) for some of these that relate to client requests.

Reply:
throwing InvalidOperationException for save and delete.